common/autoware_test_utils/include/autoware_test_utils/autoware_test_utils.hpp

```diff
@@ -387,6 +387,32 @@ T generateTrajectory(
   return traj;
 }
 
+template <>
+inline PathWithLaneId generateTrajectory<PathWithLaneId>(
+  const size_t num_points, const double point_interval, const double velocity,
+  const double init_theta, const double delta_theta, const size_t overlapping_point_index)
```

Comment:
It's nice if you can add docstring as well.

Reply:
Since the original function already has the docstring, we can ignore this comment.